pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -385,6 +410,49 @@
   farsi_page_compliance_heading: "Compliance In Support Of Farsi Translations"
   farsi_page_compliance_desc: "As a U.S. nonprofit, Code.org is subject to laws regarding sanctions with Iran. After consulting with U.S. legal counsel experienced in the Iranian Sanctions and Translations Regulations (ITSR), Code.org believes that it may fund, prepare, and distribute the Farsi Translations of CS Curriculum in the United States and elsewhere around the world, including within Iran. The ITSR provides an exemption for “information and informational materials” (the IIM Exemption) and Code.org believes that this exemption will fully shield its funding, preparation, and distribution of the Farsi Translations and thus enable its Farsi Translations effort to proceed in full compliance with U.S. economic sanctions requirements."
 
+  teach_page_top_heading: "Teach computer science & ignite possibilities"
+  teach_page_top_desc_01: "Tap into your students' full potential with Code.org's free Computer Science (CS) curriculum."
+  teach_page_top_desc_02: "Join the global community of over 2 million educators empowering the next generation of innovators, problem solvers, and digital citizens! Code.org supports teachers the whole way with professional learning, comprehensive curricula, and support when you need it."
+  teach_page_why_teach_cs_heading: "Why teach computer science with Code.org?"
+  teach_page_why_teach_cs_block_heading_01: "Build foundational CS skills"
+  teach_page_why_teach_cs_block_desc_01: "Like reading, writing, or math, computer science is foundational to other pursuits. It teaches problem solving and critical thinking, and  enables students to actively participate in an increasingly digital society."
```

Comment:
Nit: there's an extra space between "and" and "enables" in the second sentence

Reply:
Eagle eyes! 🦅